src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -2769,6 +2769,47 @@ GUI_TEST_CLASS_DEFINITION(test_7616) {
     CHECK_SET_ERR(documents.contains(initialCoiNwkDocument), "Expected initial tree document to be present in the project and re-used in MSA editor");
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7623) {
+    GTLogTracer logTracer;
+
+    // Select "Tools>Workflow Designer"
+    GTUtilsWorkflowDesigner::openWorkflowDesigner(os);
+
+    // Open "Trim and аlign Sanger reads" sample
+    class Scenario : public CustomScenario {
+    public:
+        void run(HI::GUITestOpStatus& os) {
+            // Expected state: "Trim and Align Sanger Reads" dialog has appered
+            QWidget* wizard = GTWidget::getActiveModalWidget(os);
+            const QString expectedTitle = "Map Sanger Reads to Reference";
+            const QString actualTitle = wizard->windowTitle();
+            CHECK_SET_ERR(expectedTitle == actualTitle, QString("Wizard title is incorrect: expected '%1', got '%2'").arg(expectedTitle).arg(actualTitle));
```

Comment:
Remove this check: ```WizardFiller(os, "Map Sanger Reads to Reference"``` guarantees that the scenario is triggered for the correct dialog.

Reply:
OK